algorithms/Community/speaker-listener_label_propagation/tg_algo_slpa.yml

```diff
@@ -0,0 +1,17 @@
+---
+# @           : this file
+# {x}         : the directory of file x
+# &x          : any file which matches pattern x
+# **         : any depth of arbitrary directories
+# *           : wildcard character
+# [x|y...|z]  : file/directory x, y, or z
+
+  algorithm:
+    name: Speaker-Listener Label Propagation
+    filename: "tg_slpa.gsql"
+    sha_id: ed6ea869749977cc0f3df71225d7325fb81c9767
+    description: The Speaker-listener Label Propagation Algorithm (SLPA) is a variation of the Label Propagation algorithm that is able to detect overlapping communities. The main difference between LPA and SLPA is that each node can only hold a single label in LPA while it is allowed to possess multiple labels in SLPA.
+    dependencies:
+      ExprFunctions:
```

Comment:
@wyatt-joyner-tg why do we still have the expr function dependency? Hasn't we already put all expr functions to a global file?

Reply:
> @wyatt-joyner-tg do we have any algorithm with a sub query?
Yeah, I will include that today.

The inclusion of the `ExprFunctions` or `ExprUtil` blocks indicate that this file needs the global UDF. Some queries don't have any reliance on that file.